charts/sda-svc/values.yaml

```diff
@@ -1,8 +1,11 @@
 # Default values for SDA services.
-# This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
-global:
+image:
+  repository: "ghcr.io/neicnordic/sensitive-data-archive"
+  tag: "v0.0.44"
```

Comment:
In consistent use of tag, at some places tags are used as`v0.0.44-postgres`, and here it is used without the suffix. This makes suffix such as `-pipeline` needs to be added when getting the image name. Better to be consistent. e.g. 
```
tag:  "v0.0.44-pipeline"
```